docs/resources/control-flow/backend-logic/backend-query/backend-query.md

```diff
@@ -276,3 +275,44 @@ conflict with others. Failing to do so may keep on updating the common *lastCach
 
 :::
 
+### Clear Query Cache [Action]
```

Comment:
Maybe this should be h2?

Reply:
I thought to put it under H3 to make it look more organized since its parent topic is 'Backend Query Caching' only.